src/index.js

```diff
@@ -311,61 +311,104 @@ class EventsCoreAPI {
    * Create a webhook or journal registration
    *
    * @param {string} consumerOrgId Consumer Org Id from the console
-   * @param {string} integrationId integration Id from the console
+   * @param {string} projectId Project Id from the console
+   * @param {string} workspaceId Workspace Id from the console
    * @param {object} body Json data contains details of the registration
    * @returns {Promise<object>} Details of the webhook/journal registration created
    */
-  createWebhookRegistration (consumerOrgId, integrationId, body) {
+  createRegistration (consumerOrgId, projectId, workspaceId, body) {
     const headers = {}
     const requestOptions = this.__createRequest('POST', headers, JSON.stringify(body))
-    const url = this.__getUrl(`/events/organizations/${consumerOrgId}/integrations/${integrationId}/registrations`)
+    const url = this.__getUrl(`/events/${consumerOrgId}/${projectId}/${workspaceId}/registrations`)
     const sdkDetails = { requestOptions: requestOptions, url: url }
     return this.__handleRequest(url, requestOptions, sdkDetails, codes.ERROR_CREATE_REGISTRATION)
```

Comment:
why we pass the `url` and `requestOptions` to this method, these are already contained in the `sdkDetails` ?

Reply:
i fixed it. thanks!